<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>641</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Files to be send:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="dateProcessing"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>LTspice Windows version:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Sharepoint location</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditWindowsVersion"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditLTspiceDevelopment">
       <property name="text">
        <string>C:/Users/ergue/LTspiceDevelopment</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>LTspice Mac Version:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>LTspiceDevelopment directory</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBoxSharePoint">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string>C:/Users/GErgueta/Analog Devices, Inc/New Models - New devices</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Processing date:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Files to process:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditMacVersion"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEditFilesToProcess">
       <property name="text">
        <string>C:/Users/ergue/LTspiceDevelopment/Models/ToProcess</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEditFilesToSend">
       <property name="text">
        <string>C:/Users/ergue/LTspiceDevelopment/Models/ToSend</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDevices">
    <property name="title">
     <string>Devices</string>
    </property>
    <addaction name="separator"/>
    <addaction name="getAndZip"/>
    <addaction name="separator"/>
    <addaction name="actionRemoveZipFiles"/>
    <addaction name="separator"/>
    <addaction name="check"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="process"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="generateToProcess"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="generateToSend"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="transferToAkamai"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <addaction name="menuDevices"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="getAndZip">
   <property name="text">
    <string>Get and Zip</string>
   </property>
  </action>
  <action name="check">
   <property name="text">
    <string>Check</string>
   </property>
  </action>
  <action name="process">
   <property name="text">
    <string>Process</string>
   </property>
  </action>
  <action name="generateToProcess">
   <property name="text">
    <string>Generate ToProcess</string>
   </property>
  </action>
  <action name="generateToSend">
   <property name="text">
    <string>Generate ToSend</string>
   </property>
  </action>
  <action name="transferToAkamai">
   <property name="text">
    <string>Transfer to Akamai</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRemoveZipFiles">
   <property name="text">
    <string>Remove Zip files</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
